website/content/en/highlights/2023-11-07-0-34-0-upgrade-guide.md

```diff
@@ -31,3 +32,9 @@ been removed from the Enterprise configuration. Instead of `region`, `site` shou
 #### Removal of Deprecated `component_name` Metric Tag {#deprecated-component-name}
 
 The deprecated `component_name` tag has been removed from all internal metrics. Instead the `component_id` tag should be used.
+
+#### Blackhole sink no longer reports by default {#blackhole-sink-reporting}
+
+The `blackhole` sink no longer reports events processed every second by default. Instead this
+behavior can be opted into by setting `print_interval_secs` to `1` (or any other integer). This
+change was made due to users being suprised that this sink generates output by default.
```

Comment:
```suggestion
change was made due to users being surprised that this sink generates output by default.
```

Reply:
Thanks! Fixed in [c0ea983](https://github.com/vectordotdev/vector/pull/18963/commits/c0ea9830ba75c01b70d441eabd4acf9096a2db1b)